### PR TITLE
[Code Coverage] `PNAConv` and `DegreeScalerAggregation`

### PR DESCRIPTION
**Pull request by zechengz**
PNAConv test coverage

### CODE REVIEW

File: test/nn/conv/test_pna_conv.py [hunk 1]
```diff
@@ -36,6 +36,16 @@ def test_pna_conv():
         assert torch.allclose(jit(x, adj.t()), out, atol=1e-6)
 
 
+def test_pna_divide_input():
+    x = torch.randn(4, 16)
+    edge_index = torch.tensor([[0, 0, 0, 1, 2, 3], [1, 2, 3, 0, 0, 0]])
+    deg = torch.tensor([0, 3, 0, 1])
+    conv = PNAConv(16, 32, aggregators, scalers, deg=deg, divide_input=True,
```
Comment (lightaime) [inline test/nn/conv/test_pna_conv.py:43]: Should we add `divide_input` to `__repr__` and test `str(conv)`?

File: test/nn/conv/test_pna_conv.py [hunk 2]
```diff
@@ -36,6 +36,16 @@ def test_pna_conv():
...
+                   train_norm=True)
```
Comment (lightaime) [inline test/nn/conv/test_pna_conv.py:44]: We can move `train_norm` test in [nn/aggr/test_scaler.py](https://github.com/pyg-team/pytorch_geometric/blob/master/test/nn/aggr/test_scaler.py)

File: torch_geometric/nn/conv/pna_conv.py [hunk 3]
```diff
@@ -192,17 +193,16 @@ def __repr__(self):
...
             d = degree(data.edge_index[1], num_nodes=data.num_nodes,
                        dtype=torch.long)
-            max_degree = max(max_degree, int(d.max()))
-        # Compute the in-degree histogram tensor
-        deg_histogram = torch.zeros(max_degree + 1, dtype=torch.long)
-        for data in loader:
-            d = degree(data.edge_index[1], num_nodes=data.num_nodes,
-                       dtype=torch.long)
-            deg_histogram += torch.bincount(d, minlength=deg_histogram.numel())
+            d_bincount = torch.bincount(d, minlength=deg_histogram.numel())
```
Comment (lightaime) [inline torch_geometric/nn/conv/pna_conv.py:201]: Awesome! Like it. We can also split this fix in a separate PR.

**Review by lightaime — COMMENTED** [review]
Great work!! Leave a few minor comments.

File: test/nn/conv/test_pna_conv.py (hunk 1 above)
Comment (zechengz) [reply test/nn/conv/test_pna_conv.py:43]: I am also not sure about that. It seems for other convs usually we don't add this kind of bool args to `__repr__`

File: test/nn/conv/test_pna_conv.py (hunk 2 above)
Comment (zechengz) [reply test/nn/conv/test_pna_conv.py:44]: Sure!

File: torch_geometric/nn/conv/pna_conv.py (hunk 3 above)
Comment (zechengz) [reply torch_geometric/nn/conv/pna_conv.py:201]: I see, I will split the PR.

**Review by lightaime — APPROVED** [review]
LGTM!

**Review by rusty1s — APPROVED** [review]

File: test/nn/conv/test_pna_conv.py (hunk 1 above)
Comment (lightaime) [reply test/nn/conv/test_pna_conv.py]: We do bool args sometime like `SignedConv`. Maybe @rusty1s could provide some suggestions on when we should add bool args to `__repr__`.

File: test/nn/conv/test_pna_conv.py [hunk 4]
```diff
@@ -12,16 +13,17 @@
 ]
 
 
-def test_pna_conv():
+@pytest.mark.parametrize("divide_input", [True, False])
```
Comment (rusty1s) [inline test/nn/conv/test_pna_conv.py:16]:
```suggestion
@pytest.mark.parametrize('divide_input', [True, False])
```

File: test/nn/conv/test_pna_conv.py (hunk 1 above)
Comment (rusty1s) [reply test/nn/conv/test_pna_conv.py]: I don't have a strong opinion. Usually, I prefer to only add important args to `__repr__` since otherwise the output of `print(torch.nn.Module(...))` becomes hard to parse.

Comment (lightaime) [reply test/nn/conv/test_pna_conv.py]: Cool. `divide_input` is not that important. Let's not add it.
